src/components/Login/EmailVerificationForm.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+
+import Trans from 'next-translate/Trans';
+import useTranslation from 'next-translate/useTranslation';
+import { useSWRConfig } from 'swr';
+
+import FormBuilder from '../FormBuilder/FormBuilder';
+
+import styles from './CompleteSignupForm.module.scss';
+import ResendEmailSection from './ResendEmailSection';
+
+import { completeSignup, postRequest } from 'src/utils/auth/api';
+import { makeUserProfileUrl, makeVerificationCodeUrl } from 'src/utils/auth/apiPaths';
+import ErrorMessageId from 'types/ErrorMessageId';
+import { RuleType } from 'types/FieldRule';
+import FormField, { FormFieldType } from 'types/FormField';
+
+type EmailVerificationFormProps = {
+  emailFormField: FormField;
+};
+const EmailVerificationForm = ({ emailFormField }: EmailVerificationFormProps) => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const { mutate } = useSWRConfig();
+  const [email, setEmail] = useState();
+  const { t } = useTranslation('common');
+  const requestVerificationCode = async (emailToVerify) => {
+    return postRequest(makeVerificationCodeUrl(), { email: emailToVerify });
```

Comment:
We need to have a function that does the actual API call instead of using `postRequest` directly